packages/server-wallet/src/wallet/store.ts

```diff
@@ -455,101 +444,79 @@ export class Store {
     return LedgerRequest.getPendingRequests(ledgerChannelId, tx || this.knex);
   }
 
+  /**
+   * Add a new state into the database.
+   *
+   * @param channelId - the channel ID the state refers to
+   * @param state - the new signed state to be added
+   * @param tx - an existing database transaction context
+   *
+   * This function serves as both a data validator (ensure the state is valid) and
+   * an efficient operator on the database.
+   */
   async addSignedState(
     channelId: string,
-    maybeChannel: Channel | undefined,
-    wireSignedState: WireSignedState,
+    wireState: WireSignedState,
```

Comment:
why?

Reply:
Just shortening the code — not attached to this